R/texreg.R

```diff
@@ -12,7 +12,8 @@ screenreg <- function(l, file = NULL, single.row = FALSE,
     override.ci.up = 0, omit.coef = NULL, reorder.coef = NULL, 
     reorder.gof = NULL, ci.force = FALSE, ci.force.level = 0.95, ci.test = 0, 
     groups = NULL, custom.columns = NULL, custom.col.pos = NULL, 
-    column.spacing = 2, outer.rule = "=", inner.rule = "-", ...) {
+    custom.coef.map = NULL, column.spacing = 2, outer.rule = "=", 
+    inner.rule = "-", ...) {
```

Comment:
Can you group the `custom.coef.map` argument together with the other `custom...` arguments in the function head? Perhaps after `custom.coef.names` would be a good place. Also in the other two functions. Also, don't forget to add this to the `plotreg` function.